src/locale/pages/organizeCampaigns/en.yml

```diff
@@ -1,15 +1,16 @@
 linkGroup:
-  public: Public Page
-  settings: Edit Settings
+    public: Public Page
+    settings: Edit Settings
 noManager: No Campaign Manager
 events: Events
+tasks: Tasks
 indefinite: Indefinite timeline
 calendarView: See all in calendar
-mobilization: 
-  heading: Mobilization and outreach (none configured)
-  copy: Organizing a phone bank to mobilize your organization will increase participation manifold
-  create: Create call assignment
+mobilization:
+    heading: Mobilization and outreach (none configured)
+    copy: Organizing a phone bank to mobilize your organization will increase participation manifold
+    create: Create call assignment
 feedback:
-  heading: Feedback and Surveys (none configured)
-  copy: Collecting feedback (during phonebanks or standalone) can help you work more efficiently
-  create: Create survey
\ No newline at end of file
+    heading: Feedback and Surveys (none configured)
+    copy: Collecting feedback (during phonebanks or standalone) can help you work more efficiently
+    create: Create survey
```

Comment:
The .editorconfig is configured for 4 spaces but many yml files have two :woman_shrugging: